WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -66,8 +62,8 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 			})
 			.DisposeWith(disposables);
 
-		disposables.Add(Disposable.Create(() => RxApp.MainThreadScheduler.Schedule(async () => await _qrReader.StopScanningAsync())));
+		disposables.Add(Disposable.Create(() => RxApp.MainThreadScheduler.Schedule(async () => await _qrReader.StopAsync(_cancellationToken))));
```

Comment:
Do you want to cancel the Stop? It means that you are not stopping at all. CancellationToke.None is the good one here. Usually you should not add cancellation to operations those are Closing, Stopping, Disposing things.